.github/workflows/hermes-bot-deploy.yaml

```diff
@@ -0,0 +1,59 @@
+name: github hermes-bot deploy
+
+on:
+  push:
+    paths:
+      - .github/workflows/github-bots/hermes-deploy.yaml
+      - bots/hermes/github/**
+    branches:
+      - main
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+      defaults:
+        run:
+          working-directory: bots/hermes/github
+      steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install virtualenv
+          make install-deps
+      - name: Run unit tests
+        run: |
+          make test
+      - name: Setup env variables
+# multistring env variables check this out
+# https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#multiline-strings
+#
+
+        run: |
+          echo 'SCP_FILES=scraper/github/build.tar' >> $GITHUB_ENV
+          echo 'ARCHIVE=build.tar' >> $GITHUB_ENV
+          echo 'SCP_TARGET<<EOF' >> $GITHUB_ENV
+          cat ./deploy/target.txt >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          tar -cvf build.tar $(find ./* -maxdepth 0 | grep -vf ./deploy/excluded_files.txt)
+      - name: Copy file via scp
+        uses: appleboy/scp-action@master
+        with:
+          host: ${{ secrets.DROPLET_HOST }}
+          username: ${{ secrets.DROPLET_DEPLOY_USERNAME }}
```

Comment:
can you use and create secrets for your expiremental droplet and use them as this `EXP_DROPLET_DEPLOY_USERNAME` instead of `DROPLET_DEPLOY_USERNAME`

Reply:
You can create this new `secrets` here https://github.com/osscameroon/osscameroon-website/settings/secrets/actions
and you can have more informations about that here https://docs.github.com/en/actions/reference/encrypted-secrets